include/nighthawk/common/version.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "absl/strings/str_cat.h"
+
+namespace Nighthawk {
+
+const int MAJOR_VERSION{0};
+const int MINOR_VERSION{3};
+
+class Globals {
```

Comment:
Curious - why wrap it in a class?

Considering its generic name - won't this attract more possibly unrelated methods?